src/_style.scss

```
@@ -329,3 +346,84 @@ $apple-focus-black: $apple-black;
     }
   }
 }
+
+.large-screen-background {
+  background-color: $white;
+  background-image: -webkit-linear-gradient(-11deg, $primary 50%, $white 50%);
```

Change the angle to `-10deg` and percentages to 45%.
```suggestion
  background-image: -webkit-linear-gradient(-10deg, $primary 45%, $white 45%);
```

---

Also, stick to one function for consistency. Use `linear-gradient()` and use the appropriate angle for it.